Source/Fields.H

```diff
@@ -18,6 +18,11 @@
 
 namespace warpx::fields
 {
+    /** Unique identifiers for WarpX scalar and vector fields.
+     *
+     * These are implemented as @see amrex::MultiFab (one or one per component "direction", respectively) and stored in the
+     * @see ablastr::fields::MultiFabRegister .
+     */
     AMREX_ENUM(FieldType,
```

Comment:
Maybe some hackery to `Docs/Doxyfile` can help Doxygen to pick this up...? I tried redefining `AMREX_ENUM`, but the VA_ARGS make it trickier than other macros.

Reply:
Found a solution to define and expand this macro.